microsoft-edge/webview2/concepts/webview-features-flags.md

```diff
@@ -1,95 +1,158 @@
 author: MSEdgeTeam
 ms.author: msedgedevrel
 ms.topic: conceptual
 ms.service: microsoft-edge
 ms.subservice: webview
-ms.date: 1/31/2024
+ms.date: 02/12/2024
 ---
+# WebView2 browser flags
 
-# WebView2 Browser Arguments
-One of the ways you can interact with Microsoft Edge WebView2 and impact behavior is by passing browser flags to WebView2. Generally, the flags are owned by both Chromium and Edge and are useful for testing early features and diagnosing issues. We do not recommend apps to use these in production as they can be removed or altered at any time and are not supported long term. Additionally, note that Chromium flags are not owned or controlled by Microsoft Edge so Edge does not have control over when/how the flags are removed or altered.
- 
-There are multiple ways you can set browser arguments. You can use browser args in your local device environment or through code. For testing features and diagnosing issues we recommend using browser arguments in your local device environment via setting the WEBVIEW2_BROWSER_ARGUMENTS environment variable or via reg keys. You can also set it programmatically by passing the browser arguments to the AdditionalBrowserArguments API. Note if you do it programmatically, you should remove the flags in code before shipping your app to avoid accidentally shipping the flags in production. Here is a list of some of the flags we have been seeing in the wild. 
+One of the ways you can interact with Microsoft Edge WebView2 and impact behavior is by passing browser flags (arguments, or switches) to WebView2.  These browser flags are useful for testing forthcoming features, and for diagnosing issues.  
+
+> [!WARNING]
+> Apps in production shouldn't use WebView2 browser flags, because these flags might be removed or altered at any time, and aren't necessarily supported long-term.
+
+Generally, the flags are owned<!-- todo: "are supported by"?--> by both the Chromium<!-- todo: do we actually mean the Google Chrome browser? --> browser engine and by Microsoft Edge.  Chromium flags are not owned or controlled by Microsoft Edge, so Microsoft Edge doesn't have control over when or how the flags are altered in their behavior or removed.
+
+You can set browser flags in your local device environment, or set browser flags programmatically through code.  Both approaches are described below.
+
+
+<!-- ====================================================================== -->
+## Setting browser flags in your local device environment
+
+For testing forthcoming features or diagnosing issues, we recommend using browser flags in your local device environment, via setting the `WEBVIEW2_BROWSER_ARGUMENTS` environment variable or via registry keys.  For more information, see `CreateCoreWebView2EnvironmentWithOptions`:
+
+##### [.NET/C#](#tab/dotnetcsharp)
+
+* [CreateCoreWebView2EnvironmentWithOptions]<!-- todo: url for .NET equivalent of the Win32 global `CreateCoreWebView2EnvironmentWithOptions`? -->
```

Comment:
@monica-ch @peiche-jessica - need url for the .NET & WinRT equivalents of the Win32 Global, `CreateCoreWebView2EnvironmentWithOptions`.

Reply:
WinRT - https://learn.microsoft.com/en-us/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2environmentoptions?view=webview2-winrt-1.0.2151.40

.NET - https://learn.microsoft.com/en-us/dotnet/api/microsoft.web.webview2.core.corewebview2environmentoptions.-ctor?view=webview2-dotnet-1.0.2277.86#microsoft-web-webview2-core-corewebview2environmentoptions-ctor(system-string-system-string-system-string-system-boolean-system-collections-generic-list((microsoft-web-webview2-core-corewebview2customschemeregistration)))